cmd/root.go

```diff
@@ -39,18 +34,16 @@ var rootCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		file := cmd.Flag("file")
 
-		if !strings.HasSuffix(file.Value.String(), ".yaml") || !strings.HasSuffix(file.Value.String(), ".yml") {
+		if !strings.HasSuffix(file.Value.String(), ".yaml") && !strings.HasSuffix(file.Value.String(), ".yml") {
 			fmt.Printf("Your runes file should be a yaml.")
-		} else if len(args) == 1 && strings.HasSuffix(file.Value.String(), ".yaml") {
-			parseYaml(args[0], file.Value.String())
+		} else if len(args) == 1 {
+			stage.GetStage(file.Value.String(), args[0])
```

Comment:
The cli needs to loop through the runes returned by GetStage and execute .Run() on each